ComposeAdvanced/app/build.gradle

```diff
@@ -42,6 +42,8 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
         useIR = true
+        freeCompilerArgs += "-opt-in=androidx.wear.compose.material.ExperimentalWearMaterialApi"
```

Comment:
Can move or split this out, but for a project adopting Wear Compose, putting these in every Composable seems overkill.  Should opt-in once.